packages/react-import-remote/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## [Unreleased] -->
 
+- Exported `useImportRemote` hook and `Status`([1634](https://github.com/Shopify/quilt/pull/1634))
```

Comment:
Everything has been released not sure if this is the proper way to add an unreleased change?

Reply:
looks right 👍